Upgrade to v10 breaks because SocketStream is no longer an exported member

### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the regression has not already been reported


### Last working version

9.0.0

### Stopped working in version

10.0.0

### Node.js version

20.8.0

### Operating system

Windows

### Operating system version (i.e. 20.04, 11.3, 10)

10

### 💥 Regression Report

```
src/server/client/websocket.ts:6:9 - error TS2305: Module '"@fastify/websocket"' has no exported member 'SocketStream'.
                                                                                                                       
import {SocketStream} from '@fastify/websocket';  
```

### Steps to Reproduce

pnpm build...

in my main file:
```
import fastifyWebsocket from '@fastify/websocket';
...
const server: FastifyInstance<Server, IncomingMessage, ServerResponse> = fastify({
	logger: {
		level: 'warn'
	},
	bodyLimit: 524288000
});
server.register(fastifyWebsocket);
...
await serverRegisterWebsocket(server);
```

in server.register.websocket.ts:
```
'use strict';

import {FastifyInstance} from 'fastify';
import {IncomingMessage, Server, ServerResponse} from 'http';

import websocket from '@/server/client/websocket';

const serverRegisterWebsocket = async (server: FastifyInstance<Server, IncomingMessage, ServerResponse>): Promise<void> => {
	server.register(async (s: any) => s.get('/ws', {websocket: true}, websocket));
};
export default serverRegisterWebsocket;
```

in server/client/websocket:
```
import {SocketStream} from '@fastify/websocket'; // <-- this is now not valid anymore 😞
...
export default (connection: SocketStream): void => { // <-- this is now not valid anymore 😞
	const {socket} = connection;
	socket.resume();

	socket.on('message', async (message: string) => {...
```


### Expected Behavior

it works as it did in v9...

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -98,6 +98,8 @@ declare namespace fastifyWebsocket {
     Logger extends FastifyBaseLogger = FastifyBaseLogger
   > extends fastify.RouteOptions<RawServer, RawRequest, RawReply, RouteGeneric, ContextConfig, SchemaCompiler, TypeProvider, Logger>, WebsocketRouteOptions<RawServer, RawRequest, RouteGeneric, ContextConfig, SchemaCompiler, TypeProvider, Logger> { }
 
+  export type WebSocket = WebSocket.WebSocket;
+
   export const fastifyWebsocket: FastifyWebsocket
   export { fastifyWebsocket as default }
 }

--- a/types/index.test-d.ts
+++ b/types/index.test-d.ts
@@ -1,8 +1,8 @@
-import fastifyWebsocket, { WebsocketHandler, fastifyWebsocket as namedFastifyWebsocket, default as defaultFastifyWebsocket } from '..';
+import fastifyWebsocket, { WebsocketHandler, fastifyWebsocket as namedFastifyWebsocket, default as defaultFastifyWebsocket, WebSocket } from '..';
 import type { IncomingMessage } from "http";
 import fastify, { RouteOptions, FastifyRequest, FastifyInstance, FastifyReply, RequestGenericInterface, FastifyBaseLogger, RawServerDefault, FastifySchema, RawRequestDefaultExpression } from 'fastify';
 import { expectType } from 'tsd';
-import { Server, WebSocket } from 'ws';
+import { Server } from 'ws';
 import { RouteGenericInterface } from 'fastify/types/route';
 import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox';
 import { Type } from '@sinclair/typebox'

